### ISSUE TITLE
Lib directory no longer exposed from redux-saga

### ISSUE THREAD

**Issue by tkow** [jfairbank/redux-saga-test-plan#219]
I updated redux-saga@v1,0.0-beta.2 in my project because it is recently released.
This release renewal publish directory structure and we can no longer use internal sagaHelpers from redux-saga repository. They use lerna and rollup and encapsulation some modules.

see below

<img width="297" alt="2018-09-09 19 02 20" src="https://user-images.githubusercontent.com/7004725/45263505-32b1a680-b465-11e8-9240-38fcc70a1da6.png">

and [here](https://github.com/redux-saga/redux-saga/tree/master/packages/core/src) for checking what they expose.

It needs some restructure this library or require redux-saga repository to expose them.

**Comment by hilkeheremans** [comment]
I've been digging into this issue, and you're right -- the largest impact is because of `asEffect` no longer being exported, but there are other, minor, issues to resolve all over the codebase.

Fortunately, I've managed to refactor redux-saga-test-plan so that it works again (see my fork), but have not yet managed to test thoroughly, nor have I updated the automated tests, so I'm reluctant to PR it at the moment. 

In addition to this, `redux-saga` master is currently ahead of the latest published beta and also contains some improvements (extra exports) that would be useful to finalize a proper PR. 

Finally, we have no guarantee the beta's API (with regard to inspection helpers etc at least) is stable, so it is probably not good timing to update test-plan yet.

Bottom line: I'd suggest to **hold back and use `beta.1` for now**. I'll update the tests in my fork soon and PR when it is appropriate.

@jfairbank Any input on this? Are you open to contributions/can you spend some time reviewing a PR when it's done? Thanks for your great library.

**Comment by janustoth** [comment]
I have the same problem

**Comment by buzz** [comment]
Any news on this?

[redux-saga v1.0.1 is out.](https://github.com/redux-saga/redux-saga/commit/876ac7bb9d049e9a0c9526b4396b3fd0364d3181)

**Comment by AugureMajoris** [comment]
Have encouter the issue this afternoon. Always the same. But there is a numerous items that were remove form redux-saga in the change 1.0.1.

**Comment by jp928** [comment]
This suppose to be fixed in the latest release.

**Comment by NickHeiner** [comment]
I'm still encountering this issue with  `redux-saga-test-plan@3.7.0` and `redux-saga@1.0.2`.

**Comment by jp928** [comment]
@NickHeiner
please update redux-saga-test-plan to 1.0.0beta-2

**Comment by davidleee** [comment]
> @NickHeiner
> please update redux-saga-test-plan to 1.0.0beta-2

shouldn't it version be 4.0.0-beta.2? Just came from another thread https://github.com/jfairbank/redux-saga-test-plan/issues/241#issuecomment-462230361

**Comment by dmbaranov** [comment]
@davidleee came to this issue from #217. Issue seems to be resolved with 4.0.0-beta.2
